pkg/options/configs/devnet.toml

```diff
@@ -12,3 +12,4 @@ users_address = "0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82"
 token_address = "0xa513E6E4b8f2a923D98304ec87F64353C4D5C853"
 mediation_address = "0x0B306BF915C4d645ff596e518fAf3F9669b97016"
 jobcreator_address = "0x9A9f2CCfdE556A7E9Ff0848998Aa4a0CFD8863AE"
+pow_address = ""
```

Comment:
```suggestion
pow_address = "0x4ed7c70F96B99c776995fB64377f0d4aB3B0e1C1"
```

Reply:
`dev` and `devnet` create the same addresses because we always reset the blockchain to deploy, so these should be the same, and you should also set the new controller address here: `controller_address = "0xa85233C63b9Ee964Add6F2cffe00Fd84eb32338f"`